geode-pulse/src/integrationTest/resources/pulse.properties

```diff
@@ -0,0 +1,23 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+# agreements. See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the License. You may obtain a
+# copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+pulse.oauth.provider=uaa
+pulse.oauth.clientId=pulse
+pulse.oauth.clientSecret=secret
+pulse.oauth.authorizationUri=http://xyz.com/uaa/oauth/authorize
+pulse.oauth.tokenUri=http://xyz.com/uaa/oauth/token
+pulse.oauth.userInfoUri=http://xyz.com/uaa/userinfo
+pulse.oauth.jwkSetUri=http://xyz.com/uaa/token_keys
```

Comment:
Someone else owns xyz.com, so we should not use it.

Reply:
will do.